static/Scripts/pizza_script2.js

```diff
@@ -0,0 +1,13 @@
+//switch from main to login page
+
+var indexClick = document.getElementsByClassName("pizza_image");
```

Comment:
The variable names are all over the place here. 
`pizza_image`& `indexClick` & `pizzaClick`

`indexClick` seems like an HTML element, so it isn't really a click is it?
maybe something like 
```
var pizzaImage = document.getElementsByClassName("pizza_image");
function pizzaClick () {
   pizzaImage.onlick = location.href = "login";
 } 
```
Do a similar improvement for the Signup button